apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx

```diff
@@ -643,6 +788,51 @@ export class DetailViewContents extends React.Component {
     return registrationLinks;
   };
 
+  renderInterview = () => {
+    let interviewFields = [];
+    [
+      {label: 'question1', id: 'question_1', value: this.state.question_1},
+      {label: 'question2', id: 'question_2', value: this.state.question_2},
+      {label: 'question3', id: 'question_3', value: this.state.question_3},
+      {label: 'question4', id: 'question_4', value: this.state.question_4},
+      {label: 'question5', id: 'question_5', value: this.state.question_5},
+      {label: 'question6', id: 'question_6', value: this.state.question_6},
+      {label: 'question7', id: 'question_7', value: this.state.question_7}
```

Comment:
Individually enumerating all of these questions is also adding tech debt and should be fixed. These should be passed as an arrary/list instead of 7 separate items.

Reply:
It's not that bad in terms of the debt added - we're shaving a few lines of code but this is a lower priority than hardcoded regional partner IDs